CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 2.15.0 (2019-06-06)
+
+- Updated omnibar placeholder accent to match the new omnibar gradient accent. (#92)
```

Comment:
I'd add "the" after "Updated." ... Also, in most of our changelogs, the number at the end links to the PR. Is it worth doing that here as well?

Reply:
For whatever reason, we've not linked the PR numbers in this repo.  I would say leave it as it for consistency, unless we're going to link all of them.

Reply:
Oh, I agree about doing all or none. Sorry if that wasn't clear. ... I can make the change if we are inclined to add links, but I need rights to edit the changelog.

Reply:
I updated all the entries in the changelog to include PR links.